data/114/190/933/3/1141909333.geojson

```diff
@@ -2,572 +2,576 @@
   "id": 1141909333,
   "type": "Feature",
   "properties": {
-    "edtf:cessation":"uuuu",
-    "edtf:inception":"uuuu",
-    "geom:area":0.0,
-    "geom:area_square_m":0.0,
-    "geom:bbox":"-116.22749,43.60859,-116.22749,43.60859",
-    "geom:latitude":43.60859,
-    "geom:longitude":-116.22749,
-    "iso:country":"US",
-    "lbl:bbox":"-116.24748987,43.5885901073,-116.20748987,43.6285901073",
-    "lbl:latitude":43.60859,
-    "lbl:longitude":-116.22749,
-    "lbl:max_zoom":18.0,
-    "lbl:min_zoom":13.5,
-    "mz:hierarchy_label":1,
-    "mz:is_current":1,
-    "mz:min_zoom":12.0,
-    "name:afr_x_preferred":[
-        "Boise"
+    "edtf:cessation": "",
+    "edtf:inception": "",
+    "edtf:superseded": "2024-05-30",
```

Comment:
Superseded into the Quattroshapes polygon feature today.

Reply:
When setting the is_current to 0 and superseding, we'd typically also set the cessation or deprecated date, too.. the superseded date is optional (though fine to keep in)

I'd suggest adding the second date before merging.

Reply:
Fixed in https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/168/commits/91ea1a458d788b396f3f6ab09c6a700c29e6f877, thanks!